modules/installation-aws-user-infra-rhcos-ami.adoc

```diff
@@ -18,48 +18,48 @@ You must use a valid {op-system-first} AMI for your Amazon Web Services
 |AWS AMI
```

Comment:
This is for 4.2, right? If yes, the boot image ami should be aligned with https://github.com/openshift/installer/blob/release-4.2/data/data/rhcos.json.

Reply:
Thank you @jianlinliu! Please let me know if you see other changes.